composer.json

```diff
@@ -39,6 +39,7 @@
         "ext-zend-opcache": "*",
         "doctrine/coding-standard": "^9.0.0",
         "mikey179/vfsstream": "^1.6.10",
+        "phing/phing": "^2.0",
```

Comment:
I would as a last suggestion set the minimum version to the right now installed version to prevent "rollback" failures: `^2.17.0`

Reply:
you're right ✌️